Lesson04/math.js

```diff
@@ -0,0 +1,18 @@
+function squareNumber() {
+    let input = prompt('Enter a number');
+    let result = 0;
+
+    result = Number(input)*Number(input);
+
+    alert(result);
+}
+
+function multiplyNumbers() {
+    let input = prompt('Enter a number');
+    let inputTwo = prompt('Enter another number')
+    let result = 0;
+
+    result = Number(input)*Number(inputTwo);
```

Comment:
Just a nitpick to spread out after the math operator

`result = Number(input) * Number(inputTwo);`